components/CreateForm.js

```diff
@@ -54,17 +74,26 @@ function CreateForm() {
       );
       return;
     }
+    const stepsWithModifiedUrls = inputSteps.map((inputStep) => {
+      return {
+        ...inputStep,
+        img: addProxyToImgUrl(inputStep.img),
+      };
+    });
+
     const newTutorial = {
       id: nanoid(),
```

Comment:
Why create an id here when you send it to mongoDB as the next step (which will automatically set an id for the object)?

Reply:
Good point! It was left over from previous steps.